siac/walletcmd.go

```diff
@@ -61,7 +61,7 @@ By default the wallet encryption / unlock password is the same as the generated
 	}
 
 	walletLoad033xCmd = &cobra.Command{
-		Use:   "033x [filepath]",
+		Use:   "033x",
```

Comment:
I agree it makes sense to leave it off the the filepath is specified interactively. But if the filepath is not specified interactively, shouldn't it be kept?